driver-sync/src/test/functional/com/mongodb/client/unified/Entities.java

```diff
@@ -50,10 +76,12 @@
 import static com.mongodb.client.Fixture.getMongoClientSettingsBuilder;
 import static com.mongodb.client.unified.UnifiedCrudHelper.asReadConcern;
 import static com.mongodb.client.unified.UnifiedCrudHelper.asWriteConcern;
+import static java.util.Collections.synchronizedList;
 import static java.util.Objects.requireNonNull;
 import static org.junit.Assume.assumeTrue;
 
-final class Entities {
+public final class Entities {
+    private final Set<String> entityNames = new HashSet<>();
```

Comment:
I had to add this to make some of the new failure tests pass, as there is a requirement that entities have unique names even across entity types.  We had previously ignored that requirement.